code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Food-Reagents.dm

```diff
@@ -410,19 +410,412 @@ datum/reagent/consumable/eggyolk
 	description = "It's full of protein."
 	color = "#FFB500"
 
-datum/reagent/consumable/corn_starch
+datum/reagent/egg
+	name = "Egg"
+	id = "egg"
+	description = "A runny and viscous mixture of clear and yellow fluids."
+	reagent_state = LIQUID
+	color = "#F0C814"
+
+datum/reagent/egg/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(prob(5))
+		M.emote("fart")
+	..()
+	return
+
+datum/reagent/triple_citrus
+	name = "Triple Citrus"
+	id = "triple_citrus"
+	description = "A refreshing mixed drink of orange, lemon and lime juice."
+	reagent_state = LIQUID
+	color = "#23A046"
+
+/datum/chemical_reaction/triple_citrus
+	name = "triple_citrus"
+	id = "triple_citrus"
+	result = "triple_citrus"
+	required_reagents = list("lemonjuice" = 1, "limejuice" = 1, "orangejuice" = 1)
+	result_amount = 3
+	mix_message = "The citrus juices begin to blend together."
+
+datum/reagent/triple_citrus/reaction_mob(var/mob/living/carbon/M as mob, var/method=TOUCH, var/volume)
+	if(!istype(M, /mob/living/carbon))
+		return
+	if(method == INGEST)
+		M.adjustToxLoss(-rand(1,2))
+
+datum/reagent/corn_starch
 	name = "Corn Starch"
 	id = "corn_starch"
-	description = "A slippery solution."
+	description = "The powdered starch of maize, derived from the kernel's endosperm. Used as a thickener for gravies and puddings."
+	reagent_state = LIQUID
 	color = "#C8A5DC"
 
-datum/reagent/consumable/corn_syrup
+/datum/chemical_reaction/corn_syrup
+	name = "corn_syrup"
+	id = "corn_syrup"
+	result = "corn_syrup"
+	required_reagents = list("corn_starch" = 1, "sacid" = 1)
+	result_amount = 2
+	required_temp = 374
+	mix_message = "The mixture forms a viscous, clear fluid!"
+
+datum/reagent/corn_syrup
 	name = "Corn Syrup"
 	id = "corn_syrup"
-	description = "Decays into sugar."
+	description = "A sweet syrup derived from corn starch that has had its starches converted into maltose and other sugars."
+	reagent_state = LIQUID
 	color = "#C8A5DC"
 
-datum/reagent/consumable/corn_syrup/on_mob_life(var/mob/living/M as mob)
-	M.reagents.add_reagent("sugar", 3)
-	M.reagents.remove_reagent("corn_syrup", 1)
+datum/reagent/corn_syrup/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.reagents.add_reagent("sugar", 1.2)
+	..()
+	return
+
+/datum/chemical_reaction/vhfcs
+	name = "vhfcs"
+	id = "vhfcs"
+	result = "vhfcs"
+	required_reagents = list("corn_syrup" = 1)
+	required_catalysts = list("enzyme" = 1)
+	result_amount = 1
+	mix_message = "The mixture emits a sickly-sweet smell."
+
+datum/reagent/vhfcs
+	name = "Very-high-fructose corn syrup"
+	id = "vhfcs"
+	description = "An incredibly sweet syrup, created from corn syrup treated with enzymes to convert its sugars into fructose."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+
+datum/reagent/vhfcs/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.reagents.add_reagent("sugar", 2.4)
```

Comment:
If you have the ability to inject someome with fifty units of VHFCS to hit the 200u sugar glycemic overdose, you have the ability to inject them with something a little bit nastier.

Like any of the bajillion poisons goof added. I like those. Food should not be a murder tool